README.md

```diff
@@ -22,6 +22,39 @@ Require Laravel Console Dusk using [Composer](https://getcomposer.org):
 composer require nunomaduro/laravel-console-dusk
 ```
 
+The package provide a config file that allows you to configure some options.
+```php
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Laravel Console Dusk Paths
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure the name of screenshots and logs directory as you wish.
+    */
+    'paths' => [
+        'screenshots' => 'screenshots',
+        'log'         => 'log',
```

Comment:
This should contain the absolute path. Example: `'log' => 'storage_path('laravel-console-dusk/log'),`.